### PR TITLE
Magento gift card fixes

### PR DESCRIPTION
**Pull request by ofrankowska**

### CODE REVIEW

File: Magento_GiftCard/templates/product/view/type/giftcard.phtml [hunk 1]
```diff
@@ -52,11 +52,7 @@
                         type="email"
                         id="giftcard_sender_email"
                         name="giftcard_sender_email"
-                        class="
-                            validate-email
-                            required-entry
-                            input__field
-                        "
+                        class="validate-email required-entry input__field"
```
Comment (rom3k) [inline Magento_GiftCard/templates/product/view/type/giftcard.phtml:55]: Are these classes need to be unformatted?

File: Magento_GiftCard/templates/product/view/type/giftcard.phtml [hunk 2]
```diff
@@ -102,11 +98,7 @@
                         type="email"
                         id="giftcard_recipient_email"
                         name="giftcard_recipient_email"
-                        class="
-                            validate-email
-                            required-entry
-                            input__field
-                        "
+                        class="validate-email required-entry input__field"
```
Comment (rom3k) [inline Magento_GiftCard/templates/product/view/type/giftcard.phtml:101]: Same here

File: Magento_GiftCard/templates/product/view/type/giftcard.phtml [hunk 3]
```diff
@@ -133,12 +125,12 @@
...
-
-                            <?= 'data-validate="{maxlength:'
-                                . $block->escapeHtmlAttr($block->getMessageMaxLength())
-                                . ',messages:{maxlength:' . $block->escapeHtmlAttr($block->escapeJs($errorMessage))
-                                . '}}"';
-                            ?>
+                            data-validate="{
+                                maxlength: <?= $block->escapeHtmlAttr($block->getMessageMaxLength()) ?>,
+                                messages: {
+                                    maxlength: <?= $block->escapeHtmlAttr($errorMessage) ?>
```
Comment (rom3k) [inline Magento_GiftCard/templates/product/view/type/giftcard.phtml:131]: Please use `$escaper` instance.

File: Magento_GiftCard/templates/product/view/type/giftcard.phtml (hunk 1 above)
Comment (ofrankowska) [reply Magento_GiftCard/templates/product/view/type/giftcard.phtml:55]: yep. otherwise, there's an issue with validating empty input. not sure why

**Pull request merged**